saleor/shipping/migrations/0018_default_zones_countries.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.0.6 on 2020-06-05 14:35
+
+from django.db import migrations
+
+from ..utils import get_countries_without_shipping_zone
```

Comment:
Shouldn't we copy here this function instead of importing it?

Reply:
I wasn't sure but can change that.